### ISSUE TITLE
FedMsg notifications in wrong order.

### ISSUE THREAD

**Issue by voxik** [fedora-infra/bodhi#4016]
I always receive the FedMsg notification via email in wrong order. First I receive notification

~~~
bodhi - 2020-05-07 04:15:05.268774 (karma: 0)
This update has been submitted for stable by bodhi. 
~~~

followed by

~~~
bodhi - 2020-05-07 04:15:03.986720 (karma: 0)
This update can be pushed to stable now if the maintainer wishes
~~~

Judging by the timestamps, Bodhi evaluates them in the right order, so I don't understand, where they are swapped actually.

https://bodhi.fedoraproject.org/updates/FEDORA-2020-19e6451efc

**Comment by mattiaverga** [comment]
I don't think this is caused by something in Bodhi. Looking in the code the `This update can be pushed to stable now if the maintainer wishes` message is always sent before the `This update has been submitted for stable by bodhi` message.
Also, it never happens to messages that I receive.

Just to be sure, can you check the `Date` and the `Received` headers in the emails? Those are the timestamp when the message has been sent and received by the servers.

**Comment by voxik** [comment]
The emails are sent in the same second
